hail/src/main/scala/is/hail/expr/ir/streams/EmitStream.scala

```diff
@@ -144,762 +144,639 @@ object EmitStream {
     outerRegion: Value[Region],
     env: EmitEnv,
     container: Option[AggContainer],
-  ): IEmitCode = {
-
-    @nowarn("cat=unused-locals&msg=local default argument")
-    def emitVoid(
-      ir: IR,
-      cb: EmitCodeBuilder,
-      region: Value[Region] = outerRegion,
-      env: EmitEnv = env,
-      container: Option[AggContainer] = container,
-    ): Unit =
-      emitter.emitVoid(cb, ir, region, env, container, None)
-
-    def emit(
-      ir: IR,
-      cb: EmitCodeBuilder,
-      region: Value[Region] = outerRegion,
-      env: EmitEnv = env,
-      container: Option[AggContainer] = container,
-    ): IEmitCode =
-      ir.typ match {
-        case _: TStream => produce(ir, cb, cb.emb, region, env, container)
-        case _ => emitter.emitI(ir, cb, region, env, container, None)
-      }
-
-    // returns IEmitCode of SStreamConcrete
-    def produceIterator(
-      streamIR: IR,
-      elementPType: PType,
-      cb: EmitCodeBuilder,
-      env: EmitEnv,
-    ): IEmitCode = {
-      val ecb = cb.emb.genEmitClass[NoBoxLongIterator]("stream_to_iter")
-      ecb.cb.addInterface(typeInfo[MissingnessAsMethod].iname)
-
-      val fv = FreeVariables(streamIR, false, false).eval
-      val (envParamTypes, envParams, restoreEnv) = env.asParams(fv)
-
-      val isMissing = ecb.genFieldThisRef[Boolean]("isMissing")
-      val eosField = ecb.genFieldThisRef[Boolean]("eos")
-      val outerRegionField = ecb.genFieldThisRef[Region]("outer")
-
-      ecb.makeAddPartitionRegion()
-
-      var producer: StreamProducer = null
-      var producerRequired: Boolean = false
-
-      val next = ecb.newEmitMethod("next", FastSeq[ParamType](), LongInfo)
-      val ctor = ecb.newEmitMethod(
-        "<init>",
-        FastSeq[ParamType](typeInfo[Region], arrayInfo[Long]) ++ envParamTypes,
-        UnitInfo,
-      )
-      ctor.voidWithBuilder { cb =>
-        val L = new lir.Block()
-        L.append(
-          lir.methodStmt(
-            INVOKESPECIAL,
-            "java/lang/Object",
-            "<init>",
-            "()V",
-            false,
-            UnitInfo,
-            FastSeq(lir.load(ctor.mb.this_.asInstanceOf[LocalRef[_]].l)),
-          )
-        )
-        cb += new VCode(L, L, null)
+  ): IEmitCode =
+    emitter.ctx.time {
```

Comment:
Okay. 3/4 of the PR is whitespace changes to this one file, got it.

Reply:
I'm sorry :(